pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -945,8 +945,8 @@ var _ = Describe("Snapshotter", func() {
 					err := ssr.K8sClientset.Create(ctx, lease)
 					Expect(err).ShouldNot(HaveOccurred())
 
-					ssr.HealthConfig.FullSnapshotLeaseUpdateInterval.Duration = 2 * time.Second
-					go ssr.RenewFullSnapshotLeasePeriodically(FullSnapshotLeaseStopCh)
+					fullSnapshotLeaseUpdateInterval := 2 * time.Second
```

Comment:
you can declare this once and re-use it in all tests as I can see value of `fullSnapshotLeaseUpdateInterval ` is not much varying b/w each tests. but I leave it you

Reply:
Yeah I declared it once and reused. Thanks